pybaseball/statcast_pitcher.py

```diff
@@ -87,3 +88,19 @@ def statcast_pitcher_percentile_ranks(year: int) -> pd.DataFrame:
     data = pd.read_csv(io.StringIO(res.decode('utf-8')))
     # URL returns a null player with player id 999999, which we want to drop
     return data.loc[data.player_name.notna()].reset_index(drop=True)
+
+@cache.df_cache()
+def statcast_pitcher_spin_dir_comp(year: int, pitch_combo: str = "4 - Seamer / Changeup", minP: int = 100, pov: str = "Pit") -> pd.DataFrame:
+    pitches = ["4-Seamer", "Changeup", "Curveball", "Slider", "Sinker", "Cutter"]
+    pitch_combos = [' / '.join(perm) for perm in permutations(pitches, 2)]
+    if pitch_combo not in pitch_combos:
+        print(f"Not a valid pitch combo. Must include two of {pitches}, separated by ' / '.")
+        return
+    povs = ["Pit", "Bat"]
+    if pov not in povs:
+        print(f"Not a valid pov. Must be one of {', '.join(povs)}")
+        return
```

Comment:
Or an Enum?

** ducks and runs away before @schorrm throws something at me **

Reply:
@TheCleric  I was actually thinking about it

Reply:
I'm not familiar with Enums, but yeah it could be a boolean and just set the default to whatever they use on the site.

Reply:
Actually @TheCleric I misread this from email, I thought this about an enum in pitches, which is still no because users not always the smortest but could be plausible for our backend